website/source/docs/guides/datacenters.html.md

```diff
@@ -7,6 +7,7 @@ description: |-
 ---
 
 # Multiple Datacenters
```

Comment:
I think it's important to include the entire title, especially since I think people will look for the term "multiple datacenter". I'm going to change it to `Multiple Datacenters: Basic Federation with WAN Gossip`